src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryImpl.scala

```diff
@@ -246,6 +249,7 @@ class InstanceOpFactoryImpl(
 
     spec match {
       case app: AppDefinition =>
+        logger.debug(s"Launching resident app instance ${reservedInstance.instanceId} on reservation ${reservedInstance.reservation}")
```

Comment:
```suggestion
        logger.debug(s"Launching resident app ${reservedInstance.instanceId} on reservation ${reservedInstance.reservation}")
```
because `Instance.Id def toString: String = s"instance [$idString]"`